src/wp-includes/class-wp-query.php

```diff
@@ -1885,6 +1887,10 @@ public function get_posts() {
 			$q['update_post_meta_cache'] = true;
 		}
 
+		if ( ! isset( $q['post_query_cache'] ) ) {
+			$q['post_query_cache'] = false;
```

Comment:
In the future, I would like to cache the main query on a page, which should be safe to do.